lua/headlines/init.lua

```diff
@@ -42,6 +44,7 @@ M.config = {
         fat_headlines = true,
         fat_headline_upper_string = "▃",
         fat_headline_lower_string = "🬂",
+        list_marker_string = "•",
```

Comment:
IDEA: What about different markers for different list item types?

We could allow something like this:
```lua
list_markers = {
  star = "•", -- replace only star list items
}
list_markers = {
  star = "•", -- replace star list items
  default = "-", -- replace other list items
}
```

Reply:
Not sure if there is any value in that. 

E.g. markdownlint [disallows different list marker styles by default](https://github.com/DavidAnson/markdownlint/blob/main/doc/md004.md) and prettier by default changes all markers to `-`.

Reply:
However, markdownlint also had `sublist` option for this rule, so this feature might actually be worth adding